.github/workflows/ci.yml

```diff
@@ -54,6 +54,24 @@ jobs:
           filters: |
             charts:
             - charts/nginx-gateway-fabric/**/*
+            generate:
+            - go.mod
+            - go.sum
+            - Makefile
+
+        # go mod tidy can be removed once https://github.com/renovatebot/renovate/issues/12999 is implemented
+      - name: Update files for renovate
+        run: |
+          cd tests && go mod tidy && go mod verify
+          make generate-all
```

Comment:
Is this going to run from the `tests` directory since we just changed to that directory?

Reply:
yes that's the one that doesn't get updated for indirect dependencies, when there's an update only in the main `go.mod`

Reply:
I've swapped the commands